src/main/java/org/example/Main.java

```diff
@@ -48,6 +48,5 @@ public static void main(String[] args) {
         } catch (SQLException e) {
             throw new RuntimeException(e);
```

Comment:
I don't know what should be here, but Alex will say that it's not a good practice to throw exception without any without processing. Maybe here is better to show some kind of error message for user like ("Error. Please contact with support team") and log detailed info.